TaniaGayosso/HackerRank_August25/IfElse.java

```diff
@@ -0,0 +1,29 @@
+package challenges.one;
+
+import java.util.Scanner;
+
+public class IfElse {
+
+	private static final Scanner scanner = new Scanner(System.in);
+
+	public static void main(String[] args) {
+
+		final int NUMBER = scanner.nextInt();
```

Comment:
NUMBER is a variable (read if from console), not a constant, so it doesn't need to be final

Reply:
Ok, thank you.